OrbitClientProtos/capture_data.proto

```diff
@@ -100,3 +100,9 @@ message TimerInfo {
   uint64 timeline_hash = 11;
   repeated uint64 registers = 12;
 }
+
+// libprotobuf-mutator needs a single proto with all the data
+message CaptureSerializerFuzzerInfo {
+  CaptureInfo capture_info = 2;
+  repeated TimerInfo timers = 3;
```

Comment:
Should `CaptureHeader header = 1;` be the first line?
or if not the message probably should be
```
CaptureInfo capture_info = 1;
repeated TimerInfo timers = 2;
```

Reply:
Good point. I tried with the header there, but that does not give any benefit. I will change the numbers. This can be easily changed later on, since there are no compatibility requirements.

Reply:
done.